### ISSUE TITLE
java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter: Error explained by comment but not reflected in ReadMe latest version

### ISSUE THREAD

**Issue by conorRed** [sangria-graphql/sangria-relay#46]
```
java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter
at sangria.relay.util.Base64$.encode(Base64.scala:15) 
```

Occurs because of Java version problems explained in a comment in util.Base64
`// Java 7 solution. Use java.util.Base64 in Java 8`
This causes the above error with Java 10 and sangria-relay 1.4.2 (as recommended on readMe)

I think sangria relay has been updated since this, is this just that the ReadMe has not been updated yet?

If not, how do I change to using `java.util.Base64` as recommended?

**The current fix is to add  “javax.xml.bind” % “jaxb-api” % “2.3.0”  as a dependency** 
https://hadoopandspark.wordpress.com/2018/08/22/noclassdeffounderror-datatypeconverter/

**Comment by conorRed** [comment]
I understand that the maintained sangria-relay is at https://github.com/sangria-graphql-org/sangria-relay but I couldn't submit an issue there

**Comment by nickhudkins** [comment]
Howdy @conorRed this is resolved by #37 and we hope to have a new release out that'll clear this up. Pardon the delay here :) It's been a hell of a year for Sangria :(
